components/alert/alert-toast.js

```diff
@@ -0,0 +1,183 @@
+import '../button/button-icon.js';
+import '../button/button-subtle.js';
+import '../colors/colors.js';
```

Comment:
I don't believe `button-icon`, `button-subtle` or `colors` are used directly by the toast alert?